src/serac/numerics/equation_solver.cpp

```diff
@@ -398,8 +418,9 @@ class TrustRegion : public mfem::NewtonSolver {
     double subspace_energy = computeEnergy(g, hess_vec_func, sol);
 
     if (print_options.iterations || print_options.warnings) {
+      double leftval = leftvals.size() ? leftvals[0] : 1.0;
       mfem::out << "Energy using subspace solver from: " << base_energy << ", to: " << subspace_energy << " / "
-                << energy_change << ".  Min eig: " << leftvals[0] << std::endl;
+                << energy_change << ".  Min eig: " << leftval << std::endl;
```

Comment:
It looks like the minimum eigenvalue estimate will be reported as 1.0 when no min eigenvalues estimates have been requested.  I'm fine with this for now, since it's really an inconsistent request from the user, but it is misleading. An option to consider is to check if `leftvals.size() == 0` once at the beginning of the step and echo a warning saying that the min eigenvalue will be reported as 1.0 in this case.